apple/internal/aspects/swift_static_framework_aspect.bzl

```diff
@@ -129,7 +139,7 @@ swift_library dependency with no transitive swift_library dependencies.\
                     generated_header = swiftinfo.transitive_generated_headers.to_list()[0]
 
             swiftdocs[arch] = swiftinfo.transitive_swiftdocs.to_list()[0]
-            swiftinterfaces[arch] = swiftinfo.transitive_swiftinterfaces.to_list()[0]
+            swiftinterfaces[arch] = propagated_swiftinterfaces[0]
```

Comment:
Hrm I guess this kinda change has to apply for all things accessed on `swiftinfo`